process/factory/metachain/interceptorsContainerFactory.go

```diff
@@ -75,6 +76,9 @@ func NewInterceptorsContainerFactory(
 	if check.IfNil(store) {
 		return nil, process.ErrNilStore
 	}
+	if sizeCheckDelta > 0 {
```

Comment:
does it mean that even if correct messages have been marshalised, if verified after unmarshal/marshal they will have some slight difference in size?

Reply:
That depends on the actual marshaller used, for instance json with and without indentation